cluster/ubuntu/util.sh

```diff
@@ -14,55 +14,65 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# A library of helper functions that each provider hosting Kubernetes must implement to use cluster/kube-*.sh scripts.
+# A library of helper functions that each provider hosting Kubernetes must
+# implement to use cluster/kube-*.sh scripts.
 set -e
 
-SSH_OPTS="-oStrictHostKeyChecking=no -oUserKnownHostsFile=/dev/null -oLogLevel=ERROR"
+MASTER=
+MASTER_IP=
+NODE_IPS=
 
-MASTER=""
-MASTER_IP=""
-NODE_IPS=""
+
+BASE_DIR="$KUBE_ROOT/cluster/ubuntu"
+
+SSH_OPTS="\
+ -oStrictHostKeyChecking=no\
+ -oUserKnownHostsFile=/dev/null\
+ -oLogLevel=ERROR\
+"
 
 # Assumed Vars:
 #   KUBE_ROOT
 function test-build-release {
   # Make a release
-  "${KUBE_ROOT}/build/release.sh"
+  "$KUBE_ROOT/build/release.sh"
```

Comment:
Sure. I was trying to make the file consistent, there are quite a few inconsistencies around. It would be awesome to have a linter and some linting guidelines, I'd be happy to make sure all my patches pass a certain lint config :)

I'll try to make my changes follow as much of the guideline as possible. Thanks!